NEWS

```diff
@@ -59,6 +59,8 @@ included in the vX.Y.Z section and be denoted as:
 
 4.0.4 -- May, 2020
 -----------------------
+- Add checks to avoid conflicts with a libevent library shipped with LSF.
```

Comment:
So this PR is more than just checks.  It actually changes our dependency from `-levent` to `-levent_core`.  That SHOULDn't matter, but perhaps we should mention that?  If you agree, I can update if you'd like.

Reply:
I think you're right.  adding now.